src/vs/languages/css/common/parser/cssParser.ts

```diff
@@ -955,6 +976,34 @@ export class Parser {
 		return null;
 	}
 
+	public _parseCssVariable(): nodes.Variable {
+		var node = <nodes.Variable> this.create(nodes.Variable);
+		if (!this.accept(scanner.TokenType.CssVariableName)) {
+			return null;
+		}
+		return <nodes.Variable> node;
+	}
+
+	public _parseVariableExpression(): nodes.Node {
+		if (!this.accept(scanner.TokenType.Var)) {
+			return null;
+		}
+
+		var node= this.createNode(nodes.NodeType.VariableExpression);
+		if (!this.accept(scanner.TokenType.ParenthesisL)) {
+			return this.finish(node, errors.ParseError.LeftParenthesisExpected);
+		}
+
+		if (!node.addChild(this._parseCssVariable())) {
+			return this.finish(node, errors.ParseError.VariableValueExpected);
+		}
+
+		if (!this.accept(scanner.TokenType.ParenthesisR)) {
```

Comment:
The var syntax also allows extra parameters (default values).
see https://drafts.csswg.org/css-variables/#using-variables, example 10.

As mentioned I'd just use the regular _parseFunction for that.